projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/package-lock.json

```diff
@@ -464,6 +464,7 @@
       "version": "7.20.7",
       "resolved": "https://registry.npmjs.org/@babel/plugin-proposal-async-generator-functions/-/plugin-proposal-async-generator-functions-7.20.7.tgz",
       "integrity": "sha512-xMbiLsn/8RK7Wq7VeVytytS2L6qE69bXPB10YCmMdDZbKF4okCqY74pI/jJQ/8U0b/F6NrT2+14b8/P9/3AMGA==",
+      "deprecated": "This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-async-generator-functions instead.",
```

Comment:
Please check if we can switch to the recommended plugins for all the babel ones that are deprecated.

Reply:
I will do so in separate PR